ui/page/components/consensus_list.go

```diff
@@ -0,0 +1,162 @@
+package components
+
+import (
+	"image/color"
+
+	"gioui.org/layout"
+	"gioui.org/text"
+
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/load"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+var canVote bool
+
+type ConsensusItem struct {
+	Agenda     dcrlibwallet.Agenda
+	VoteButton decredmaterial.Button
+}
+
+func AgendasList(gtx C, l *load.Load, consensusItem *ConsensusItem) D {
+	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+	return layout.UniformInset(values.MarginPadding16).Layout(gtx, func(gtx C) D {
+		agenda := consensusItem.Agenda
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layoutAgendaStatus(gtx, l, consensusItem.Agenda)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layoutAgendaDescription(gtx, l, agenda)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layoutAgendaID(gtx, l, agenda)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layoutAgendaVotingPreference(gtx, l, agenda)
+			}),
+			layout.Rigid(func(gtx C) D {
+				return layoutAgendaVoteAction(gtx, l, consensusItem)
+			}),
+		)
+	})
+}
+
+func layoutAgendaStatus(gtx C, l *load.Load, agenda dcrlibwallet.Agenda) D {
+	lbl := l.Theme.H5(agenda.AgendaID)
+	lbl.Font.Weight = text.SemiBold
+
+	var statusLabel decredmaterial.Label
+	var statusLabelColor color.NRGBA
+	var statusIcon *decredmaterial.Icon
+	var backgroundColor color.NRGBA
+
+	switch agenda.Status {
+	case "Finished":
+		statusLabel = l.Theme.Label(values.MarginPadding14, agenda.Status)
+		statusLabelColor = l.Theme.Color.GreenText
+		statusIcon = decredmaterial.NewIcon(l.Icons.NavigationCheck)
+		statusIcon.Color = statusLabelColor
```

Comment:
```suggestion
		statusIcon.Color = l.Theme.Color.Green500
```

lets separate icon and text colors.